components/phpunit_bridge.rst

```diff
@@ -0,0 +1,129 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
+=============================
+
+    The PHPUnit Bridge component provides utilities to report trigerred errors,
+    legacy tests and usage of deprecated code.
+
+It comes with the following features:
+
+* Enforce a consistent `C` locale (It forces applications to use the default language for output)
+* Auto-register `class_exists` to load Doctrine annotations (when used);
+* Print a user deprecation notices summary at the end of the test suite;
+* Display the stack trace of a deprecation on-demand.
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>`
+  (``symfony/phpunit-bridge`` on `Packagist`_); as a dev dependency
+* Use the official Git repository (https://github.com/symfony/phpunit-bridge).
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+Once the component installed, it automatically registers a
+`PHPUnit event listener`_. This listener simply registers a `PHP error handler`_
+called `DeprecationErrorHandler`. After running your PHPUnit tests again, you
+will have a similar report:
+
+.. image:: /images/components/phpunit_bridge/report.png
+
+The summary includes:
+
+* **Unsilenced** reports deprecation notices that were triggered without the recommended @-silencing operator;
```

Comment:
how do you do that? If I do:

```
* Beginning of my line
second part
```

It will fire a warning:

```
WARNING: Bullet list ends without a blank line; unexpected unindent.
```